src/Tickets/Seating/Orders/Controller.php

```diff
@@ -218,7 +231,7 @@ public function filter_seat_tab_title( $title, $post_id, $page_type ): string {
 	 * @return void
 	 */
 	public function register_seat_reports() {
-		$this->container->make( Seats_Report::class )->hook();
+		$this->seats_report->hook();
```

Comment:
Let's not do this, remove the `hook` method from the `Seats_Report` class.
The Seats Report is hooking on `admin_menu`, hook **this controller** to that action and delegate the rendering to the Seats Report.
This way the controller `unregister` method will **actually** unregister from `admin_menu` as well.